playbooks/roles/lets-encrypt/templates/response.yaml.j2

```diff
@@ -0,0 +1,13 @@
+"acme-enter-email": "{{ streisand_admin_email }}"
+"acme-agreement:https://letsencrypt.org/documents/LE-SA-v1.1.1-August-1-2016.pdf": true
+"acmetool-quickstart-choose-server": https://acme-v01.api.letsencrypt.org/directory
+"acmetool-quickstart-choose-method": redirector
+# This is only used if "acmetool-quickstart-choose-method" is "webroot".
+"acmetool-quickstart-webroot-path": "/var/www/foo/bar/.well-known/acme-challenge"
```

Comment:
If we don't use this mode can we remove this line? It seems strange to have a `/var/www/foo/bar` config line if the folder doesn't exist.

Reply:
@cpu I'll remove that line.